.github/workflows/build-deploy-dev.yml

```diff
@@ -5,42 +5,133 @@ on:
     types: [opened, reopened, synchronize, closed]
     branches:
       - main
+  workflow_dispatch:
+    inputs:
+      bumb:
+        description: "The type of semantic version increment to make. By default it is 'prerelease'."
+        required: false
+        default: "prerelease"
+        type: choice
+        options:
+          - major
+          - premajor
+          - minor
+          - preminor
+          - patch
+          - prepatch
+          - prerelease
+      preId:
+        description: "Identifier to be used to prefix version increments. By default it is 'prerelease' for pull request event and workflow dispatch will follow the last release tag prefix, if any."
+        required: false
+        type: choice
+        options:
+          -
```

Comment:
Why there's an empty option?

Reply:
its the prefix, for example, the last tag was `v0.1.1-prelerease-7`.  and now we want to build and release create a release with a new tag with a `major` bump and we don't wish to add any prefix in the tag, then don't need to select the preId and it will create a new tag like `v1.0.0` if we are setting a preid for eg, `premajo` then it will create a tag like `v1.0.0-premajor.0`. If you are having a hard tie to follow, then we can have quick sync up